pyfiglet/__init__.py

```diff
@@ -46,6 +48,8 @@
 
 RESET_COLORS = b'\033[0m'
 
+SHARED_DIRECTORY = '%APPDATA%/pyfiglet/' if sys.platform == 'win32' else '/usr/local/share/pyfiglet/'
```

Comment:
I'm expecting to find something which expands `%APPDATA%` into an actual path - or does windows interpret that during the `open()` function? Not a windows user so I can't test it.

Reply:
Good spot!  I got a bit trigger happy on the PR.  meant to test that before submitting...  Looks like it needs an explicit conversion.  WIll change.